e2e/components/dialog/create-from-template-dialog.ts

```diff
@@ -49,7 +49,7 @@ export class CreateFromTemplateDialog extends GenericDialog {
   }
 
   async isValidationMessageDisplayed(): Promise<boolean> {
-    return (await this.validationMessage.isPresent()) && (await this.validationMessage.isDisplayed());
+    return (await this.validationMessage.isPresent()) && (this.validationMessage.isDisplayed());
```

Comment:
Maybe second condition needs await too?
```suggestion
    return (await this.validationMessage.isPresent()) && (await this.validationMessage.isDisplayed());
```